paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutUIScreenshotTest.kt

```diff
@@ -10,6 +11,7 @@ import org.junit.Rule
 import org.junit.Test
 import org.mockito.kotlin.mock
 
+@OptIn(ExperimentalEmbeddedPaymentElementApi::class)
```

Comment:
Probably better to just opt in earlier. This is all internal stuff, we don't need to propagate it all the way out.

Reply:
The `PaymentMethodVerticalLayoutUI` function this is calling already is opted in, but because this method has RowStyle in the constructor it needs the annotation as well